5calls/app/src/main/java/org/a5calls/android/a5calls/model/AccountManager.java

```diff
@@ -34,10 +32,10 @@ public enum AccountManager {
     private static final String KEY_NOTIFICATION_DIALOG_SHOWN = "prefsKeyNotificationDialog";
     private static final String KEY_CALLER_ID = "prefsKeyCallerID";
     private static final String KEY_REVIEW_DIALOG_SHOWN = "prefsKeyReviewDialog";
-    private static final String KEY_LOCATION_NAME = "prefsKeyLocationName";
     private static final String KEY_NEWSLETTER_PROMPT_DONE = "prefsKeyNewsletterPrompt";
     private static final String KEY_NEWSLETTER_SIGNUP_DONE = "prefsKeyNewsletterSignup";
     public static final String KEY_USER_NAME = "prefsKeyUserName";
+    public static final String KEY_SCRIPT_TEXT_SIZE = "prefsKeyScriptTextSize";
```

Comment:
optional: I've been asked to put units on constants before, so this would be something like KEY_SCRIPT_TEXT_SIZE_SP. Up to you, this is probably fine as-is.